execution-timeline/build.gradle.kts

```diff
@@ -0,0 +1,65 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
```

Comment:
Please modify the deploy script to deploy this also in maven

Reply:
Done